static/app/utils/profiling/hooks/useDifferentialFlamegraphModel.tsx

```diff
@@ -0,0 +1,93 @@
+import {useMemo} from 'react';
```

Comment:
This would have been duplicated, so I created a reusable hook. The idea is to pass it a raw before/after response and generate the differential flamegraph

Reply:
Could also consider making this a regular helper function and let the caller wrap it in `useMemo`.  But if you want more granular control over how things are memoed, then this is fine too 👍